tests/php/Control/Email/EmailTest/templates/SilverStripe/Control/Tests/Email/EmailTest/EmailSubClass.ss

```diff
@@ -7,7 +7,7 @@
 
 <div class="body">
     <h1>Email Sub-class</h1>
-	$EmailContent
+	$EmailContent.RAW
```

Comment:
I've changed this to .RAW here for easier unit testing, but not on the actual default Email.ss template due to the remote chance of HTML injection on the default template. If users want to put .RAW on their own custom email templates, leave that decision up to end users.